indico/web/forms/fields/privacy.py

```diff
@@ -0,0 +1,23 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2022 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from wtforms import ValidationError
+
+from indico.util.i18n import _
+from indico.web.forms.fields.simple import JSONField
+from indico.web.forms.widgets import JinjaWidget
+
+
+class IndicoPrivacyPolicyURLsField(JSONField):
+    widget = JinjaWidget('forms/privacy_policy_urls_widget.html', single_kwargs=True, single_line=True)
+
+    def pre_validate(self, form):
+        for item in self.data:
```

Comment:
Mostly a matter of preference, but I think this is a bit nicer since it only has to get the len() once (there's no meaningful difference as len is O(1) in Python anyway)

```python
if not all(x.get('url') for x in self.data):
    raise ..
if len(self.data) > 1 and not all(x.get('title') for x in self.data):
    raise ..
```